clis/ph-cli/src/commands/uninstall.ts

```diff
@@ -0,0 +1,138 @@
+import fs from "node:fs";
+import { execSync } from "node:child_process";
+import { Command } from "commander";
+
+import { CommandActionType } from "../types.js";
+import {
+  getProjectInfo,
+  getPackageManagerFromLockfile,
+  PackageManager,
+  packageManagers,
+  updateConfigFile,
+  SUPPORTED_PACKAGE_MANAGERS,
+} from "../utils.js";
+
+export function uninstallDependency(
+  packageManager: PackageManager,
+  dependencies: string[],
+  projectPath: string,
+  workspace?: boolean,
+) {
+  if (!fs.existsSync(projectPath)) {
+    throw new Error(`Project path not found: ${projectPath}`);
+  }
+
+  const manager = packageManagers[packageManager];
+
+  let uninstallCommand = manager.uninstallCommand.replace(
+    "{{dependency}}",
+    dependencies.join(" "),
+  );
+
+  if (workspace) {
+    uninstallCommand += ` ${manager.workspaceOption}`;
+  }
+
+  const commandOptions = { cwd: projectPath };
+
+  execSync(uninstallCommand, {
+    stdio: "inherit",
+    ...commandOptions,
+  });
+}
+
+export const uninstall: CommandActionType<
+  [
+    string[] | undefined,
+    {
+      debug?: boolean;
+      global?: boolean;
+      workspace?: boolean;
+      packageManager?: string;
+    },
+  ]
+> = (dependencies, options) => {
+  if (options.debug) {
+    console.log(">>> command arguments", { dependencies, options });
+  }
+
+  if (!dependencies || dependencies.length === 0) {
+    throw new Error("❌ Dependency name is required");
+  }
+
+  if (
+    options.packageManager &&
+    !SUPPORTED_PACKAGE_MANAGERS.includes(options.packageManager)
+  ) {
+    throw new Error(
+      "❌ Unsupported package manager. Supported package managers: npm, yarn, pnpm, bun",
+    );
+  }
+
+  const projectInfo = getProjectInfo(options.debug);
+
+  if (options.debug) {
+    console.log("\n>>> projectInfo", projectInfo);
+  }
+
+  const isGlobal = options.global || projectInfo.isGlobal;
+  const packageManager =
+    options.packageManager || getPackageManagerFromLockfile(projectInfo.path);
+
+  if (options.debug) {
+    console.log("\n>>> uninstallDependency arguments:");
+    console.log(">>> packageManager", packageManager);
+    console.log(">>> dependencies", dependencies);
+    console.log(">>> isGlobal", isGlobal);
+    console.log(">>> projectPath", projectInfo.path);
+    console.log(">>> workspace", options.workspace);
+  }
+
+  try {
+    console.log("installing dependencies 📦 ...");
```

Comment:
The log should be "uninstalling dependencies"

Reply:
Added fix here: [028ad40](https://github.com/powerhouse-inc/powerhouse/pull/965/commits/028ad40805b0c6c0299b4bddccb6c95933c63d0e)